game/src/lights.rs

```diff
@@ -19,7 +19,7 @@ impl Lights {
                 // TODO(cristicbz): Remove this restriction.
                 assert!(self.lights.len() < 255);
                 self.lights.push(light_info.clone());
-                (self.lights.len() - 1)
```

Comment:
is the `_usize` suffix necessary? it seems like it should work without it

Reply:
@cristicbz it's really not, just my preference if it's problem I can redo it

Reply:
@cristicbz hi, I removed usize suffix